### DISCUSSION TITLE
If we have more than 20 items in side navigation which is not fit in screen how we can show them.

### DISCUSSION THREAD

**Discussion by UX-DattatrayBhosale** [iTwin/iTwinUI-react#789] [Q&A]
I am using side navigation component for plantSight and we have facility to add custom items in side navigation.
now in normal laptop screen we can see 10 to 12 item's vertically if user add more then 20 item how we can show them?
Note: almost every item has sub menu to access multiple services. 

<img width="1407" alt="Screenshot 2022-08-19 at 1 49 54 PM" src="https://user-images.githubusercontent.com/111564131/185578428-46de4097-39ae-433a-9930-0079eedb069c.png">

**Comment by mayank99** [comment]
Hi @UX-DattatrayBhosale, have you tried making the list scrollable using `overflow: auto`?

**Reply by UX-DattatrayBhosale** [reply]
@mayank99 we don't want scrollbar to shown for navigation. imagine i have 2 scrollbar one for side navigation and  second for content area. i don't think scroll bar is good option.

**Reply by veekeys** [reply]
@UX-DattatrayBhosale so what do you suggest? Do you have any mockup available?
